src/project/projecttypes.h

```diff
@@ -210,6 +210,14 @@ struct Template
 
 using Templates = QList<Template>;
 
+//struct RecentFile {
+//    io::path_t path;
+//    QString displayName;
+//};
+
+using RecentFile = io::path_t;
+using RecentFilesList = std::vector<RecentFile>;
```

Comment:
Spoiler alert: the plot unravels in (currently still only the comment in) https://github.com/musescore/MuseScore/pull/17764